README.md

```diff
@@ -37,7 +37,7 @@ For other environments, you can choose between installing via pip or directly fr
 
 #### GPU
 
-If you want to run sockeye on a GPU you need to make sure your version of MXNet contains the GPU code. Depending on your
+If you want to run sockeye on a GPU you need to make sure your version of Apache MXNet contains the GPU code. Depending on your
```

Comment:
I thought one needs to mention some form of the word "incubating" when talking about an incubating project.